.github/workflows/localstack.yml

```
@@ -0,0 +1,53 @@
+name: Deploy Domain Protect into LocalStack
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - "**/*.py"
+      - "**/*.tf"
+      - "**/*.txt"
+      - ".github/workflows/localstack.yml"
+
+jobs:
+  localstack:
+    name: localstack
+    runs-on: ubuntu-22.04
```

Might as well use latest since it uses 24.04
```suggestion
    runs-on: ubuntu-latest
```